README.md

```diff
@@ -1,6 +1,10 @@
-kerl [![CircleCI](https://circleci.com/gh/kerl/kerl.svg?style=svg)](https://circleci.com/gh/kerl/kerl)
+kerl [![GitHub Actions CI][ci-img]][kerl] [![GitHub Actions Lint][lint-img]][kerl]
 ====
 
+[kerl]: https://github.com/kerl/kerl
+[ci-img]: https://github.com/kerl/kerl/workflows/ci/badge.svg
```

Comment:
This should hopefully be seen in 🟢 after a test action is run for the first time.